### PR TITLE
Append title, dateTime and link

### LINKED ISSUE #21: Save date, webpage title and link with text
Concatenate date and webpage link and title with selected text and save it to paper. Remember, we are using markdown format.
Title should be saved as heading or in bold text.

### PR DESCRIPTION
**Pull request by Insiyaa**
This fixes #21.

### CODE REVIEW

**Comment by Insiyaa** [comment]
This also fixes #22

**Comment by AdityaHirapara** [comment]
@Insiyaa, Attach some screenshots of paper with added text.

**Comment by Insiyaa** [comment]
screenshot from 2018-06-22 23-52-49

File: js/paper.js [hunk 1]
```diff
@@ -112,6 +114,10 @@ function saveToPaper(token, paperId, text) {
         chrome.storage.sync.set({Revision : new_rev.revision});
       } 
     }
-    xhr.send(text);
+    var today = new Date();
+    var date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
+    var time = today.getHours() + ":" + today.getMinutes() + ":" + today.getSeconds();
+    var dateTime = date+' '+time;
```
Comment (AdityaHirapara) [inline js/paper.js]: Add space around '+'

**Pull request merged**